src/styles/Location.css

```diff
@@ -0,0 +1,80 @@
+.location {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.flex {
+  display: flex;
+}
+
+.location--contain {
+  margin-bottom: 10px;
+  justify-content: space-between;
+  max-width: 150px;
+}
+
+.num {
+  font-family: "Raleway", sans-serif;
+}
+
+.text {
+  font-family: "Lato", sans-serif;
+}
```

Comment:
These classes could definitely have better names! I would think of these as modifiers in terms of BEM, eg `location__result--number` or `location__result--description` would be a cleaner way of doing this